distro.py

```diff
@@ -30,6 +30,7 @@
     "holidays.countries",
     "holidays.financial",
     "isodate",
+    "iXBRLViewerPlugin",
```

Comment:
This doesn't seem to work [on windows](https://github.com/austinmatherne-wk/Arelle/actions/runs/7154147100)